rosidl_generator_cpp/rosidl_generator_cpp/__init__.py

```diff
@@ -114,10 +114,14 @@ def msg_type_to_cpp(type_):
             (type_.pkg_name, type_.type)
 
     if type_.is_array:
-        if type_.array_size is None:
+        if not type_.array_size:
```

Comment:
I guess that `type_.array_size` cannot be `0`? Is this change important to the functionality of the bounded array or just another way to check for `None`?